docs/manual/roles.md

```diff
@@ -0,0 +1,61 @@
+# Roles
+
+Roles can be used to give a set of permissions on an object to users or groups of users.
+
+## Team
+
+Team allow to assign a role to a group of users. A team can be created by anyone, the creator will be the admin by default. Each member will have the permissions associated with the team.
+
+## Roles in Team
```

Comment:
TODO: replace all of of this by a simple table with role name and attached permissions.